articles/metadata/index.md

```diff
@@ -13,17 +13,17 @@ Auth0 allows you to store **metadata**, or data related to each user that has no
 An authenticated user can modify data in their profile's `user_metadata`, but not in their `app_metadata`.
 :::
 
-### How to Read, Create or Edit Metadata
+## How to Read, Create or Edit Metadata
 
 You can manage your metadata using [Rules](/rules/metadata-in-rules) or the [Auth0 APIs](/metadata/management-api).
 
-::: warning
-Note that `app_metadata` fields are not [searchable](/api/management/v2/user-search). For `user_metadata`, you can only search for profile information: `name`, `nickname`, `given_name`, `family_name`.
+::: note
+Note that `app_metadata` fields are not [searchable](/api/management/v2/user-search). For `user_metadata`, you can only search for profile information, such as `name`, `nickname`, `given_name`, or `family_name`.
 :::
```

Comment:
@jeffreylees I'm going to turn this back to a warning since we want to put emphasis on it. We had some major issues due to these searches this week